plugins/Jobs.cpp

```diff
@@ -402,6 +402,8 @@ void BedrockJobsCommand::process(SQLite& db) {
         //     - jobPriority - High priorities go first (optional, default 500)
         //     - unique - if true, it will check that no other job with this name already exists, if it does it will
         //                return that jobID
+        //     - overwrite - if true, and unique is true, it will overwrite the existing job with the new jobs data
```

Comment:
```suggestion
        //     - overwrite - Only applicable when submit is is true. When set to true it will overwrite the existing job with the new jobs data
```